integration_tests/steps/jira_issues_tests.py

```diff
@@ -258,3 +258,16 @@ def device_statistics(context):
                     assert len(list(output_dict[house][device]["min_" + stats_name])) == 48
                     assert len(list(output_dict[house][device]["max_" + stats_name])) == 48
     assert counter == 12
+
+
+@then('trades happen when the load seeks energy')
+def trades_happen(context):
+    from pendulum import today
+    from d3a.constants import TIME_ZONE
```

Comment:
Can we get rid of this lazy imports?

Reply:
Can you tell me the reason of importing inside the step. As far as I see all the [constants](https://github.com/gridsingularity/d3a/blob/master/integration_tests/environment.py#L51) are reset after every scenario. Let me know if I have misunderstood.